src/inc/msquic.h

```diff
@@ -146,6 +146,8 @@ typedef enum QUIC_CREDENTIAL_FLAGS {
     QUIC_CREDENTIAL_FLAG_REVOCATION_CHECK_CACHE_ONLY            = 0x00040000, // Windows only currently
     QUIC_CREDENTIAL_FLAG_INPROC_PEER_CERTIFICATE                = 0x00080000, // Schannel only
     QUIC_CREDENTIAL_FLAG_SET_CA_CERTIFICATE_FILE                = 0x00100000, // OpenSSL only currently
+    QUIC_CREDENTIAL_FLAG_SET_MULTIPLE                           = 0x00200000, // Schannel only currently
+    QUIC_CREDENTIAL_FLAG_SET_ALLOWED_CERT_ALGS                  = 0x00400000, // Schannel only currently
```

Comment:
We've been trying to not have abbreviations. But I don't think _CERTIFICATE_ALGORITHMS will fit. Maybe _CERTIFICATE_ALGS?